### PR TITLE
Woo Express: Add plan features and feature groups

### PR DESCRIPTION
**Pull request by markbiek**
Related to #75383

## Proposed Changes

This adds all of the Woo Express-specific plan features and feature groups.

This is part of several PRs related to plan features and the plan comparison grid:
* #75918 (this PR) which adds all of the plan features and groups.
* #75769 which enables the plan comparison grid for Woo Express.
* #75883 which adds footnotes to the plan comparison grid.

## Testing Instructions

This Slack conversation (p1681499986012049-slack-C0Q664T29) has a link to a spreadsheet with all of the features.

 * Inspect the code and verify that the correct features exist and are listed under the correct groups.

## Pre-merge Checklist

- [x] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [x] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: client/lib/plans/features-list.tsx [hunk 1]
```diff
@@ -2168,6 +2194,139 @@ export const FEATURES_LIST: FeatureList = {
 			i18n.translate( 'Keep an administrative eye on activity across your site.' ),
 	},
 	/* END: 2023 Pricing Grid Features */
+
+	/* START: Woo Express Features */
+	[ FEATURE_WOOCOMMERCE_STORE ]: {
+		getSlug: () => FEATURE_WOOCOMMERCE_STORE,
+		getTitle: () => 'WooCommerce store',
```
Comment (sixhours) [inline client/lib/plans/features-list.tsx]: Do all these `getTitle` values need to be wrapped in translate functions? I see some are and some are not.

**Review by sixhours — APPROVED** [review]
I think this is good! I went through line by line and compared with the spreadsheet (although there are a *lot* of things to compare so I worry there's a lot of room for human error on my part -- I wonder if it makes sense to write some tests to ensure we're including the right features with the right plans/groups? Or does it matter so much, since these are constants and we're not manipulating the data very much?)

I noticed the spreadsheet has a typo:
Screen Shot 2023-04-18 at 3 11 08 PM

**Comment by markbiek** [comment]
>  I wonder if it makes sense to write some tests to ensure we're including the right features with the right plans/groups?

~We don't seem to have tests for any of the other constants but this is probably as a good a time as any to add them!~

Based on our Slack discussion, we've decided tests don't really make sense. p1681844113068389-slack-C0Q664T29

File: client/lib/plans/features-list.tsx (hunk 1 above)
Comment (markbiek) [reply client/lib/plans/features-list.tsx]: They definitely should be wrapped in translate functions. Thanks!

**Comment by markbiek** [comment]
Screenshot for translation:

image

**Pull request merged**